docs/content/guides/dagster/local_to_prod_tutorial.mdx

```diff
@@ -0,0 +1,393 @@
+---
+title: Development to Production Workflow | Dagster
+description: This guide walks through how to transition your code from local development to production
+---
+
+# Development to Production Workflow
```

Comment:
This title feels very uninspired to me. I'll be thinking of ideas for more interesting titles, but would love some suggestions too